mangaki/mangaki/tests/test_refresh_from_anidb.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import responses
+from django.conf import settings
+from django.test import TestCase
+
+from django.core.urlresolvers import reverse
+from django.contrib.auth import get_user_model
+from django.contrib import admin
+
+from mangaki.models import Work, Category
+from mangaki.utils.anidb import AniDB, client
+
+
+class RefreshFromAniDBTest(TestCase):
+    @staticmethod
+    def read_fixture(filename):
+        with open(os.path.join(settings.TEST_DATA_DIR, filename), 'r', encoding='utf-8') as f:
+            return f.read()
+
+    @staticmethod
+    def add_to_responses(filename):
+        responses.add(
+            responses.GET,
+            AniDB.BASE_URL,
+            body=RefreshFromAniDBTest.read_fixture('anidb/'+filename),
+            status=200,
+            content_type='application/xml'
+        )
+
+    def setUp(self):
+        self.user = get_user_model().objects.create_superuser(username='test', password='test', email='email@email.email')
+        client.__init__('test', 1)
```

Comment:
The issue is that a `RuntimeError('AniDB API is not available!')` is raised since no `ANIDB_CLIENT` and `ANIDB_VERSION` are provided in the settings. It works perfectly when they are provided, even if fake values are given for both.

I guess one solution would be to add fake values to the `.circle_mangaki_settings.ini` so that tests don't fail ? ([here](https://github.com/mangaki/mangaki/pull/433/commits/93cee196f0b0246c0433b87c695e3d5cc11bca2e)'s the commit where I added this `__init__` call, that's what made the test pass on GitHub)